tests/gentropy/conftest.py

```diff
@@ -84,6 +84,11 @@ def mock_colocalisation(spark: SparkSession) -> Colocalisation:
         .withColumnSpec("h4", percentNulls=0.1)
         .withColumnSpec("log2h4h3", percentNulls=0.1)
         .withColumnSpec("clpp", percentNulls=0.1)
+        .withColumnSpec(
+            "colocalisationMethod",
+            percentNulls=0.0,
```

Comment:
I don't know too much about the details, but going down from 0.1 to 0.0 is expected?

Reply:
`colocalisationMethod` contains the method we used to get the result. It should never be null.